[MTM-55154] crl proper error code fix (#1908)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-10-20-294-0-crl-proper-error-code.md

---
date: ""

> title: CRL entry response status with correct error code 

Reviewer: Rephrase: _Fixed the http response code for wrong CRL entries_

Reviewer 2: I would suggest `Fixed the http response code for missing fields in CRL entries`

> title: Fixed the http response code for missing fields in CRL entries

Reviewer 3: ```suggestion
title: Correct HTTP response code sent on missing fields in CRL entries
```

> title: Correct HTTP response code sent on missing fields in certificate revocation list (CRL) entries

Reviewer 3: ```suggestion
title: Correct HTTP response code sent on missing fields in CRL (Certificate Revocation List) entries
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: q3kclF6pO
    label: Authentication
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-55154
version: 10.20.294.0
---

> Wrong error code being shown for a missing field in CRL entry payload is now fixed.

Reviewer: _Wrong http response code was sent when there are missing fields in CRL entry. This is fixed now._

> Wrong http response code was sent when there are missing fields in CRL entry. This is fixed now.

Reviewer 3: ```suggestion
A wrong HTTP response code was sent when fields were missing in the CRL entry. This behavior has been fixed now.
```

> Wrong http response code was sent when there are missing fields in CRL entry. This is fixed now.

Reviewer 3: What is CRL? Please do not abbreviate without context.

Author: Please check now
